### ISSUE TITLE
Tab order incorrect on first Profile Tab

### ISSUE THREAD

**Issue by j2tausig** [mitodl/micromasters#1418]
Steps to Reproduce:
Log onto MicroMasters.
Create a Profile on edX.
Select the Program field.
Navigate to the next field (Given Name) using the Tab.

Expected Behavior:
One Tab click should bring the user to the Given Name Field.
Actual Behavior:
It takes 3 Tab clicks to navigate to the Given Name Field.
Additionally, if a user is on the Given Name field and attempts to tab back up to the Program Field, that action does not work at all.

NOTE:  The transition between those 2 fields has an issue, all the other fields on the first Profile Tab look good.

**Comment by rhysyngsun** [comment]
This actually appears to be an issue with `material-ui` in that the `<SelectField/>` is not tab-able. A ticket was opened on this 7 days ago: https://github.com/callemall/material-ui/issues/5498. Additionally, there's an accessibility issue here where a user can't select an option from `<SelectField/>` via the keyboard. I'll see if I can fix that for `<PersonalTab/>` but we also need to patch it elsewhere and/or figure out and contribute an upstream patch.

**Comment by rhysyngsun** [comment]
@pdpinch 

I've spent what I feel is a reasonable amount of time on this (1-2h) and not made much progress. Putting `tabindex="0"` on the `<SelectField/>` allows it to be tabbed into, but not out of. From the tickets I've read and a few StackOverflow questions, this seems to be a known issue with that component. Additionally, the component is not really accessible anyway because `<SPACE>` does not open the select as one would expect.

I checked react-select and it does not appear to have this issue. Perhaps we can fix this after #1659 is merged and we have a reasonable styled version of that available. @aliceriot any thoughts on how easy that should be based on your work?

**Comment by aliceriot** [comment]
Yea react-select deals with tabbing a lot more sanely.

**Comment by aliceriot** [comment]
So I think we should be good :)
